net/yaaw/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yaaw
+PKG_VERSION:=2016-05-28
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Hsing-Wang Liao <kuoruan@gmail.com>
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/binux/yaaw.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=c28cde2252b707fe0661693f78c5e5e3beee09a1
+PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.gz
```

Comment:
Please consider using `$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz` as the name of tarball.  This also applies to the pr about `webui-aria2`.